### DISCUSSION TITLE
v1.0.5: Better error handling

### DISCUSSION THREAD

**Discussion by wneessen** [wneessen/go-hibp#26] [Announcements]
This releases overhauls the error handling of the different APIs. Aside of fixing #22, in this release we reworked the general error handling. Errors are no defined via `errors.New()` and therefore can be checked via `errors.Is()` accordingly. General code cleanup was also performed. Check the PRs for all details.

## What's Changed
* Fix missing HTTP response return in PastedAccount method. by @wneessen in https://github.com/wneessen/go-hibp/pull/23
* Fix/24 verify and overhaul the error handling of the different apis by @wneessen in https://github.com/wneessen/go-hibp/pull/25

Thanks to @chandru89new for finding the error and raising the error issue.

**Full Changelog**: https://github.com/wneessen/go-hibp/compare/v1.0.4...v1.0.5

<hr /><em>This discussion was created from the release <a href='https://github.com/wneessen/go-hibp/releases/tag/v1.0.5'>v1.0.5: Better error handling</a>.</em>

**Comment by chandru89new** [comment]
Happy to have been of some small help. Just wanted to re-iterate that on behalf of the team at Steampipe, thank you for all your effort on this SDK!
